primitives/runtime/src/lib.rs

```diff
@@ -358,6 +376,18 @@ impl Verify for AnySignature {
 	}
 }
 
+impl BatchVerify for AnySignature {
```

Comment:
Please remove. This is wrong.

`AnySignature` can not be batched.

`try_from` will always return a valid `Signature` as long as the `Signature` has the correct length. However, with `AnySignature` we first need to check `sr25519` and if that fails, we need to check `ed25519`.